omod/src/main/java/org/bahmni/module/hip/web/service/ValidationService.java

```diff
@@ -10,6 +11,7 @@
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Service;
 
+import java.util.Date;
 import java.util.List;
 @Service
 @Qualifier("validationService")
```

Comment:
We don't require a ValidationService for this purpose. You can use a custom validator driven by annotation with little bit of spring help. Or just leverage the default date input validation (right now, that is done by the openmrs dateinput).